server/src/main/java/org/apache/iotdb/db/qp/logical/crud/QueryOperator.java

```diff
@@ -56,11 +57,31 @@
 
   private boolean ascending = true;
 
+  private Map<String, Object> props;
```

Comment:
"props" is a little general and may be confused if the original query involves other parameters/ properties, shall we rename it to "indexProps"?

Reply:
yes, I think it's ok